Examples/Python/python/acts/examples/__init__.py

```diff
@@ -285,3 +286,39 @@ def _allmods(mod, base, found):
     if not quiet and donemods:
         print("dump_args for module functions:", ", ".join(donemods))
     return alldone
+
+
+class CustomLogLevel(Protocol):
+    def __call__(
+        self,
+        override: Optional[acts.logging.Level] = None,
+        min: acts.logging.Level = acts.logging.VERBOSE,
+        max: acts.logging.Level = acts.logging.FATAL,
+    ) -> acts.logging.Level:
+        ...
+
+
+def defaultLogging(
+    s=None,
+    logLevel: Optional[acts.logging.Level] = None,
+) -> CustomLogLevel:
+    """
+    Establishes a default logging strategy for the python examples interface.
+
+    Returns a function that determines the log level in the following schema:
+    - if `overrideLevel` for `customLogLevel` is set return it
+    - if `logLevel` is set use it other wise use the log level of the sequencer `s.config.logLevel`
+    - the returned log level is bound between `min` and `max` provided to `customLogLevel`
```

Comment:
Maybe a snippet of examples would be useful here:
i.e. if you want all components to produce only warnings or more severe errors do that:...
if you want switch all components to INFO do that ...

Reply:
good point will do